packages/headless/src/database/mongodb/otp/otp.model.ts

```diff
@@ -0,0 +1,23 @@
+import { model, Schema } from 'mongoose';
+import { Otp } from 'src/entity/otp';
+
+const OtpSchema = new Schema<Otp>({
+  phone: String,
+  email: String,
+  otp: {
+    type: Number,
+    unique: true,
```

Comment:
otp can not be unique. actually you can't do that, because of the short length of otp